esp-box-3/include/bsp/esp-box-3.h

```diff
@@ -127,6 +141,12 @@ typedef struct {
     lvgl_port_cfg_t lvgl_port_cfg;
 } bsp_display_cfg_t;
 
+/**
+ * @brief BSP IMU type
+ *
+ */
+typedef icm42670_handle_t bsp_imu_t;
```

Comment:
I assume that we plan to create more `typedef some_imu_handle_t bsp_imu_t;` in other BSPs.

This will create very loose dependency, that will be very easy to break in the future.

The example still uses `icm42670_xxx` API. What is the benefit of this type then?

Reply:
I thought, that we will have same API for init for all IMU `bsp_imu_t bsp_imu_init(void)`. And I thought, that we can use anz IMU in example, but sure, there is API from specific IMU. 
Ok, it was bad idea. 

I would like see independent API for everything inside BSP. But if we really want it, there must be something like `esp_lcd_touch` or somethung like that :-(